guides/intro.md

```diff
@@ -0,0 +1,615 @@
+
+    ```mermaid
+    flowchart LR
+      subgraph elixir [Elixir App]
+        PCE1[PeerConnection 1]
+        Forwarder-- "4. add_track(track12)" -->PCE2[PeerConnection 2]
+        PCE2-- "3. {:track, track22}" -->Forwarder
+      end
+
+      subgraph P2["Peer 2 (Web Browser)"]
+        U2[User]-- "1. addTrack(track22)" -->PCW2[PeerConnection]
+        PCW2-- "6. ontrack(track12)" --> U2
+      end
+
+      PCW2-. "2. offer/answer" .->PCE2
+      PCE2-. "5. offer/answer" .->PCW2
+    ```
+
+    - assuming that we expect only _N_ tracks from Peer 2, we can use the tip above and
+    make sure that there are at least _N_ transceivers in Peer 2's PeerConnection on the Elixir side and do just a single negotiation.
+    Note that you can also add transceivers without associated track, that's what you would need to do if
+    _N_ in the diagram was greater than 1, because we only have a single track available.
+
+    ```mermaid
+    flowchart BR
+      subgraph elixir [Elixir App]
+        PCE1[PeerConnection 1]
+        Forwarder-- "2. add_transceiver(track12)" -->PCE2[PeerConnection 2]
+        PCE2-- "4. {:track, track22}" -->Forwarder
+      end
+
+      subgraph P2["Peer 2 (Web Browser)"]
+        U2[User]-- "1. addTrack(track22)" -->PCW2[PeerConnection]
+        PCW2-- "5. ontrack(track12)" --> U2
+      end
+
+      PCE2-. "3. offer/answer" .->PCW2
+   ```
+
+3. Lastly, Peer 1 also wants to receive Peer 2's tracks, so we need to add the new tracks to Peer 1's PeerConnection and perform the renegotiation there.
+
+```mermaid
+flowchart LR
+  subgraph P1["Peer 1 (Web Browser)"]
+    PCW1[PeerConnection]-- "3. ontrack(track12)" --> U1
+  end
+
+  subgraph elixir [Elixir App]
+    Forwarder-- "1. add_track(21)" -->PCE1[PeerConnection 1]
+    PCE2[PeerConnection 2]
+  end
+
+  PCE1-. "2. offer/answer" .->PCW1
+```
+
+> #### Who owns the tracks? {: .warning}
+> Each of the tracks exists only in the context of its own PeerConnection. That means even if your Elixir App forwards media from one peer to
+> another, it only takes RTP packets from a track in the first peer's PeerConnection and feeds them to another track in the second peer's PeerConnection.
+> For instance, the role of `Forwarder` in the examples above would be to forward media in such way:
+>
+> ```mermaid
+> flowchart LR
+>   subgraph Forwarder
+>     track11 -.-> track12
+>     track22 -.-> track21
+>   end
+>   PC1[PeerConnection 1] --> track11
+>   PC2[PeerConnection 2] --> track22
+>   track12 --> PC2
+>   track21 --> PC1
+> ```
+>
+> This might be a bit counterintuitive, as in reality both of the tracks `track11` and `track12` still carry the same media stream.
+
+A similar process would happen for all of the joining/leaving peers. If you want to check an actual working example, check out the
+[Nexus](https://github.com/elixir-webrtc/apps/tree/master/nexus) - our Elixir, WebRTC-based videoconferencing demo app.
```

Comment:
This sentence got broken in half

<img width="499" alt="image" src="https://github.com/user-attachments/assets/5f2635a8-513f-4837-8295-410fd47d06bd">